libs/gretty-core/src/main/groovy/org/akhikhl/gretty/ServerConfig.groovy

```diff
@@ -20,7 +20,7 @@ import groovy.transform.TypeCheckingMode
 @ToString
 class ServerConfig {
 
-  static final int RANDOM_FREE_PORT = PortUtils.RANDOM_FREE_PORT
+  static final int RANDOM_FREE_PORT = -1
```

Comment:
You could reuse the `ServerDefaults.RANDOM_FREE_PORT` variable that you've added.

Reply:
Unfortunately not, this constant moved to a shared module between the CLI runner and the servlet container runners. I do not want to drag in another module just for reusing one constant.
Besides that, now these two can also evolve independently. We might use -1 for the build DSL, and use a different sentinel value in the runner projects. A value of -1 is more of a historic fact than reasonable choice. Usually, binding the port 0 gives a 'random' free port.